server/src/messageHandler.mjs

```diff
@@ -215,7 +215,7 @@ async function handleMessage(message, userId, ws) {
   } else {
     // Handle Firebolt Method call using default defaults (from the examples in the Open RPC specification)
     logger.debug(`Returning default mock value for method ${oMsg.method}`);
-    response = stateManagement.getMethodResponse(userId, oMsg.method, oMsg.params); // Could be optimized cuz we know we want a static response
+    response = stateManagement.getMethodResponse(userId, oMsg.method, oMsg.params,ws); // Could be optimized cuz we know we want a static response
```

Comment:
take care of spacing between , and ws

Reply:
Taken care and updated. Please review it again